examples/go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/ipfs/go-block-format v0.1.2
 	github.com/ipfs/go-cid v0.4.1
 	github.com/ipfs/go-datastore v0.6.0
-	github.com/ipld/go-ipld-prime v0.20.0
+	github.com/ipld/go-ipld-prime v0.20.1-0.20230619045951-499e9b9c2024
```

Comment:
@lidel yeah, that'd be great, but I have no idea if there's an ETA. cc @rvagg

Reply:
Yeah, sorry, this is a pressing concern for me too; I wanted to squeeze one more item into go-ipld-prime (https://github.com/ipld/go-ipld-prime/pull/530) before cutting a release since the releases there are a bit more involved than most other simpler libraries and we have a bit of a backlog of items already. I'm slowly getting back into some work this week but it probably won't be until next week to get this out; I hope this is OK.